modules/model/shared/src/main/scala/gem/util/Location.scala

```diff
@@ -176,9 +174,11 @@ object Location {
         // but rounding down to make them integral. Since gapSize is at least
         // 1.0, this will always advance and never produce duplicates.
         (1 to count)
+          .view
           .scanLeft(startBd) { (sum, _) => sum + gapSize }
-          .drop(1)
-          .map { bd => fromBase10(bd.setScale(0, FLOOR).toBigInt) }(breakOut)
+          .drop(1)          
+          .map { bd => fromBase10(bd.setScale(0, FLOOR).toBigInt) }
+          .toList
```

Comment:
There's a test for this method (`Location.find`) failing: [Find should evenly space values it finds](https://github.com/ghik/silencer). I can't figure out why mostly because I don't know the domain. It doesn't seem related to the change though.

Reply:
Yeah try `.iterator` here as well, or just leave it out (in both cases).

Reply:
Changed both `.view`s to `.iterator`s. Besides working better in 2.12, it also makes more sense in these cases.